api/package.json

```diff
@@ -44,7 +44,7 @@
     "jsonwebtoken": "^9.0.0",
     "jwks-rsa": "3.1.0",
     "lodash": "^4.17.21",
-    "minio": "^7.1.0",
+    "minio": "^7.1.1",
```

Comment:
We need to use minio version v7.1.1, the latest version v1.7.3 returns an error when running tests. It doesn't block the tests, all tests still pass. I feel it's okay to keep a lower version for now. 
<img width="772" alt="294060624-b79eddc2-3abd-4999-976a-d33cbd7e0982" src="https://github.com/bcgov/nr-fom/assets/23131735/47e73843-670c-49a5-80ee-226193878541">